c2rust-ast-exporter/build.rs

```diff
@@ -273,23 +281,62 @@ impl LLVMInfo {
 
         let llvm_config = find_llvm_config();
         let llvm_config_missing = "
+        Couldn't find `llvm-config`. Make sure `llvm-config` is on $PATH then
+        re-build.
+        ";
```

Comment:
As far as I can tell, with these changes this string becomes dead. But I'm not opposed to keeping it around as a reminder to add better error handling for this situation.